.github/workflows/verify_newproject_script.yml

```diff
@@ -11,11 +11,11 @@ jobs:
     runs-on: ubuntu-latest
     timeout-minutes: 30
     steps:
-      - name: Set up JDK 11
+      - name: Set up JDK 17
```

Comment:
there is one more `- name: Set up JDK 17` in this workflow

Reply:
Updated both in [082ad3e](https://github.com/nimblehq/android-templates/pull/540/commits/082ad3e1226ef72ee4af7b9d2bd346f69afea1fc)